src/internet_identity/src/stats/event_stats.rs

```diff
@@ -389,6 +389,8 @@ fn update_aggregation<M: Memory, F>(
     let mut pruned_weight_by_key = pruned_events
         .iter()
         .filter_map(&aggregation_filter_map)
+        // zero weighted events are ignored for pruning
```

Comment:
nit: this comment is somewhat confusing when compared to the test `should_prune_zero_weighted_events` added below.  Maybe reformulate/add more info?

Reply:
I adjusted it. Is it clearer now?